projects/plugins/jetpack/extensions/blocks/ai-paragraph/edit.js

```diff
@@ -28,15 +31,15 @@ function getSuggestionFromOpenAI(
 ) {
 	const needsAtLeast = 36;
 	if ( formattedPrompt.length < needsAtLeast ) {
+		setLoadingCompletion( false );
 		setErrorMessage(
 			sprintf(
-				/** translators: First placeholder is a number of more characters we need, second is the total number we need */
+				/** translators: First placeholder is a number of more characters we need */
 				__(
-					'AI needs you to write %1$d more characters above this block. When inserted into an empty post with a title (or additionally categories selected), AI Paragraph will generate a new post for you. If you insert the AI Paragraph block following other text, it will attemp to auto-complete, but it needs at least %2$d characters of input.',
+					'Please write a longer title or a few more words in the opening preceding the AI block. Our AI model needs %1$d more characters.',
```

Comment:
I was expecting the number to update itself when I was typing elsewhere...

Reply:
Yeah but that would require that number to be updated but the render to be **NOT** automatically triggered, so it would basically be 2 `select`

Reply:
You should be able to [subscribe to data changes](https://wordpress.stackexchange.com/questions/357267/get-used-blocks-in-post-and-detect-changing) and update that value, no?

Reply:
No, it's more complicated, feel free to try to tackle that in https://github.com/Automattic/jetpack/issues/28389 !